salt/modules/ipmi.py

```diff
@@ -0,0 +1,844 @@
+# -*- coding: utf-8 -*-
+'''
+Support IPMI commands over LAN
+
+:depends: Python module pyghmi
+
+:warning: pyghmi version >= 0.6.21
+
+you can install pyghmi using:
+
+    sudo git clone https://github.com/stackforge/pyghmi.git  /usr/lib/python2.6/site-packages/
+
+The following configuration defaults can be define in the pillar:
+
+    ipmi.config:
+        api_host: 127.0.0.1
+        api_user: admin
+        api_pass: apassword
+        api_port: 623
+        api_kg: None
+
+most calls can override the api connection config defaults:
+
+    salt-call ipmi.get_user api_host=myipmienabled.system
+                            api_user=admin api_pass=pass
+                            uid=1
+'''
+
+
+IMPORT_ERR = None
+try:
+    from pyghmi.ipmi import command
+    from pyghmi.ipmi.private import session
+except Exception as ex:
+    IMPORT_ERR = str(ex)
+
+__virtualname__ = 'ipmi'
+
+
+def __virtual__():
+    return (IMPORT_ERR is None, IMPORT_ERR)
```

Comment:
ah yes the disconnect is when using salt-call. It would be nice if using the `salt` command worked.
```
salt minion ipmi.user_delete
cloud.math.xxx.ca:
    'ipmi.user_delete' is not available.
    Error loading module.ipmi: No module named pyghmi.ipmi
```